lib/libutee/tee_api_objects.c

```diff
@@ -532,8 +532,8 @@ TEE_Result TEE_RenamePersistentObject(TEE_ObjectHandle object,
 	return res;
 }
 
-TEE_Result TEE_AllocatePersistentObjectEnumerator(TEE_ObjectEnumHandle *
-						  objectEnumerator)
+TEE_Result TEE_AllocatePersistentObjectEnumerator(TEE_ObjectEnumHandle
```

Comment:
Purely subjective, it's what I usually do when facing this dilemma. I think it's a good compromise and doesn't look too ugly.